docs/sources/userguide/dockerimages.md

```diff
@@ -505,6 +505,25 @@ Let's see our new tag using the `docker images` command.
     ouruser/sinatra     devel   5db5f8471261  11 hours ago   446.7 MB
     ouruser/sinatra     v2      5db5f8471261  11 hours ago   446.7 MB
 
+## Image Digests
+
+Images that use the v2 or later format have a content-addressable identifier
+called a `digest`. As long as the input used to generate the image is
+unchanged, the digest value is predictable. To list image digest values, use
+the `--digests` flag:
+
+    $ docker images --digests | head
+    REPOSITORY                         TAG                 DIGEST                                                                     IMAGE ID            CREATED             VIRTUAL SIZE
+    ouruser/sinatra                    latest              sha256:cbbf2f9a99b47fc460d422812b6a5adff7dfee951d8fa2e4a98caa0382cfbdbf    5db5f8471261        11 hours ago        446.7 MB
+
+When pushing or pulling to a 2.0 registry, the `push` or `pull` command
+output includes the image digest. You can `pull` using a digest value. 
+
+    $ docker pull training/sinatra@0bd60042b5a8dbc7f083a8e7f5c6bc3533c24dbb085a337bccf23f4a252c36a3
+
```

Comment:
I think it would be clearer if the image and digest used here, would be the same as is shown in the example above. Then people would understand which part of the digest should be used to "pull" (ie, not include the `sha256:`)